api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -41,6 +41,16 @@ type VirtualMachineImageOSInfo struct {
 	Type string `json:"type,omitempty"`
 }
 
+// VirtualMachineImageInfo describes basic information related to the image, such as id and name. This information
+// can be used to identify an image in the content source.
+type VirtualMachineImageInfo struct {
+	// ImageID describes a unique identifier provided by the provider of this VirtualMachineImage.
+	ImageID string `json:"imageId"`
```

Comment:
To match k8s convention the JSON fields should use "ID", like "imageID" for this field.